changelogs/current.yaml

```diff
@@ -30,6 +30,9 @@ bug_fixes:
   change: |
     Fix a RELEASE_ASSERT when using :ref:`auto_sni <envoy_v3_api_field_config.core.v3.UpstreamHttpProtocolOptions.auto_sni>`
     if the downstream request ``:authority`` was longer than 255 characters.
+- area: tracing
+  change: |
+    Fix an issue where span id is missing from opentelemetry access log entries
```

Comment:
style-nit: add a '.' at the end of the line (also for comments in the source files).

Reply:
I checked through all source changes and am pretty sure that I have fixed all instances of comments that were not terminated with periods.